x/migration/keeper/morse_claimable_account.go

```diff
@@ -0,0 +1,71 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/store/prefix"
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/runtime"
+
+	"github.com/pokt-network/poktroll/x/migration/types"
+)
+
+// SetMorseClaimableAccount set a specific morseClaimableAccount in the store from its index
+func (k Keeper) SetMorseClaimableAccount(ctx context.Context, morseClaimableAccount types.MorseClaimableAccount) {
+	storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	store := prefix.NewStore(storeAdapter, types.KeyPrefix(types.MorseClaimableAccountKeyPrefix))
+	b := k.cdc.MustMarshal(&morseClaimableAccount)
+	store.Set(types.MorseClaimableAccountKey(
+		morseClaimableAccount.Address.String(),
+	), b)
+}
+
+// GetMorseClaimableAccount returns a morseClaimableAccount from its index
+func (k Keeper) GetMorseClaimableAccount(
+	ctx context.Context,
+	address string,
+
+) (val types.MorseClaimableAccount, found bool) {
+	storeAdapter := runtime.KVStoreAdapter(k.storeService.OpenKVStore(ctx))
+	store := prefix.NewStore(storeAdapter, types.KeyPrefix(types.MorseClaimableAccountKeyPrefix))
+
+	b := store.Get(types.MorseClaimableAccountKey(
+		address,
+	))
+	if b == nil {
+		return val, false
```

Comment:
Optional NIT across the whole file. Consider renaming `val` or `b` to more semantic vars related to this business logic (future proof future business logic).

Reply:
Leaving one comment but ditto everywhere